service/resource/endpoint/create_test.go

```diff
@@ -148,7 +148,7 @@ func Test_Resource_Endpoint_GetCreateState(t *testing.T) {
 		}
 	}
 	for i, tc := range testCases {
-		result, err := newResource.GetCreateState(context.TODO(), tc.Obj, tc.CurrentState, tc.DesiredState)
+		result, err := newResource.newCreateChange(context.TODO(), tc.Obj, tc.CurrentState, tc.DesiredState)
```

Comment:
The test function name should be renamed accordingly.

Reply:
Renamed and reordered.